src/wp-includes/class-wp-scripts.php

```diff
@@ -863,6 +863,18 @@ private function get_dependents( $handle ) {
 		return $dependents;
 	}
 
+	/**
+	 * Check if the strategy is valid or not.
+	 *
+	 * @param string $strategy The script strategy.
+	 * @return bool True if strategy part of strategies.
+	 */
+	private function is_valid_strategy( $strategy ) {
```

Comment:
```suggestion
	private function is_valid_strategy( $strategy = '' ) {
```

@kt-12 We never check for the presence of the `$strategy` param and assume its presence in the function body. This could lead to a warning. I suggest we default the param value.

Reply:
@10upsimon I believe since we are trying to validate a value in its true form, we should never default it to a blank string.
Currently, it's only used in `get_intended_strategy` which has a `strategy` check.